doc/MANUAL.txt

```diff
@@ -451,6 +451,14 @@ might be incorrect.
     matching the compiler name in the normal *PATH* that isn't a symbolic link
     to ccache itself.
 
+*pch_external_checksum* (*CCACHE_PCH_EXTSUM*)::
+
+    When this option is set, and ccache finds a precompiled header file,
+    ccache will look for a file with the extension ``.sum`` added
+    (e.g. ``pre.h.gch.sum``), and if found, it will hash this file instead
```

Comment:
The third and fourth backticks should be apostrophes; see [“Double quoted text”](http://asciidoc.org/asciidoc.css-embedded.html#X51) in the AsciiDoc User Guide.